example/package.json

```diff
@@ -10,10 +10,14 @@
     "build": "react-scripts build"
   },
   "dependencies": {
+    "deep-equal": "^1.0.1",
     "immutable": "^3.8.1",
+    "mapbox-gl": "^0.35.0",
+    "reduce-object": "^0.1.3",
+    "supercluster": "^2.3.0",
     "react": "^15.5.3",
     "react-dom": "^15.5.3",
-    "react-mapbox-gl": "^1.7.0",
+    "react-mapbox-gl": "..",
```

Comment:
can you keep `^1.7.0` (it is used for automatic deployment of the examples) if you want to setup a dev env you have to link react-mapbox-gl, just run `npm run link`